.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```

Comment:
If we are going down this path (which I'm fine with), lets add the following as well:

```json
  "typescript.tsdk": "node_modules/typescript/lib"
```